internal/pkg/sync/config.go

```diff
@@ -279,6 +281,7 @@ type location struct {
 	SkipTLSVerify bool           `yaml:"skip-tls-verify"`
 	AuthRefresh   *time.Duration `yaml:"auth-refresh"`
 	lastRefresh   time.Time
+	expiry        time.Time
```

Comment:
Since we have two expiry/refresh variables, we should make clear which is which, and be consistent in the naming. E.g., `ecrExpiry` and `gcrExpiry`, or `ecrLastRefresh` and `gcrLastRefresh`.

Reply:
They are referring to different points in time in relation to time.Now(). I renamed both struct fields, though, to better reflect their purpose.